content/00.front-matter.md

```diff
@@ -25,12 +30,31 @@ on {{manubot.date}}.
 
 ## Authors
 
+### Version 2.0 Authors
+
+{% for author in manubot.authors %}
+  {% if author.v2 is defined -%}
+    {% if author.orcid is defined -%}
+    [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/{{author.orcid}})
+    {%- endif %}
+    {{author.name}}<sup>{{author.affiliation_numbers | join(',')}}♠{%- if author.symbol_str is defined -%},{{author.symbol_str}}{%- endif -%}</sup>
+    {%- if not loop.last -%}, {%- endif -%}
+  {%- endif -%}
+{% endfor %}
+
+<sup>♠</sup> --- Author order for version 2.0 is currently arbitrary.<br>
```

Comment:
Should we make it alphabetical so we can give clear instructions to new Version 2.0 authors?  That may be too tricky if some authors are both v1 and v2 authors.

Reply:
I wonder if there's a way to have the v1 field become the author contributions grouping there and the v2 field become the same (instead of just true/empty). That opens up the potential to put author contributions on mouseover, and also to sort the list dynamically (say, alphabetically within contribution bands) as people contribute.

Reply:
Tracking the contribution types in the metadata file makes sense to me.  Should we save v2 author ordering and mouseover contributions for a follow up issues/pull request?

Reply:
Agree